docs/BUILD.md

```diff
@@ -1,6 +1,6 @@
 # Local setup
 
-Clarity uses NodeJS 14+ and Yarn 1.22.4+ for development, so ensure you have them installed and up to date. To find the exact
+Clarity uses NodeJS 14+ for development, so ensure you have it installed and up to date. To find the exact
```

Comment:
I think this is changing for the angular branch. I think I saw @steve-haar bump the node version to 17. 

In addition, do we want to point to the new repo and wait to merge this until after the angular code base is live on the new angular repo at `github.com/vmware-clarity/ng-clarity`?

Reply:
I don't have access to that repo yet - do we have a timeline for when we expect to migrate over there? It seems we're at a bit of a standstill for now, and if so I think it's best to merge these changes sooner rather than later to help anyone trying to contribute

Reply:
I think Steve just opened that PR but it's not merged yet. I did approve so it wouldn't block this. This can always be updated after the repo changes settle down.